Sources/BraintreePaymentFlow/BTPaymentFlowClient.swift

```diff
@@ -0,0 +1,138 @@
+import Foundation
+import AuthenticationServices
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+@objcMembers public class BTPaymentFlowClient: NSObject {
+    
+    // MARK: - Internal Properties
+    
+    var authenticationSession: ASWebAuthenticationSession?
+    
+    // MARK: - Private Properies
+    
+    private let _apiClient: BTAPIClient
```

Comment:
```suggestion
    private let apiClient: BTAPIClient
```

Reply:
This has to have a different name than `apiClient` to satisfy this [protocol requirement](https://github.com/braintree/braintree_ios/blob/7cc5fde720d2b65e3f0f3bbcfa7385c7c9f617ae/Sources/BraintreePaymentFlow/BTPaymentFlowClient.swift#L119-L121) or else swift complains with `Invalid redeclaration of 'apiClient'`.

In a follow-up PR I think we can remove that protocol requirement entirely